src/Tests/Microsoft.NET.Build.Containers.IntegrationTests/LayerEndToEndTests.cs

```diff
@@ -42,7 +42,7 @@ public void SingleFileInFolder()
         VerifyDescriptorInfo(l);
 
         var allEntries = LoadAllTarEntries(l.BackingFile);
-        Assert.True(allEntries.TryGetValue("app/", out var appEntry) && appEntry.EntryType == TarEntryType.Directory, "Missing app directory entry");
+        Assert.True(allEntries.TryGetValue("app", out var appEntry) && appEntry.EntryType == TarEntryType.Directory, "Missing app directory entry");
```

Comment:
Zip entries have trailing slashes to denote they are directory entries, but tar files don't need this suffix.
I removed it in the implementation, and updated the test accordingly.